acceptance-tests/run.sh

```diff
@@ -57,4 +59,6 @@ if [ "${RUN_SELENIUM}" == "true" ]; then
     ./runner/scripts/stop-selenium.sh
 fi
 
+./runner/scripts/stop-bitbucket-server.sh
```

Comment:
Is there anything that would make this script fail before it gets here?

Reply:
or at least need to clean it up before starting (which we do for selenium)